recipe/meta.yaml

```diff
@@ -11,10 +11,20 @@ source:
   fn: cvxopt-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/c/cvxopt/cvxopt-{{ version }}.tar.gz
   md5: d4ff0a0e51aba93f838a70302a088eea
+  patches:
+    # Fix the environment variable used for SuiteSparse includes.
+    # See this PR ( https://github.com/cvxopt/cvxopt/pull/60 ).
+    - cvxopt_PR_60.diff
+    # Add a patch that fixes the linking order and adds a missing library.
+    # See this commit
+    # ( https://github.com/cvxopt/cvxopt/commit/007c1a0ea9df9a54e9f99cacbde56a6047518375 ).
+    # Also, see this PR to show how this was backported.
+    # ( https://github.com/cvxopt/cvxopt/pull/61 )
+    - cvxopt_PR_61.diff
```

Comment:
See this commit ( https://github.com/cvxopt/cvxopt/commit/007c1a0ea9df9a54e9f99cacbde56a6047518375 ) for the upstream change. However, that did not apply cleanly. So see this PR ( https://github.com/cvxopt/cvxopt/pull/61 ) for how that was applied. It results in the same net change, but with a whitespace difference.